patch_roadmap_2.sh

```
@@ -0,0 +1,2 @@
+sed -i 's/is \*\*verified-public\*\* for `v1.13.1` as of 2026-06-21/is \*\*verified-public\*\* for `v1.14.0` as of 2026-06-25/g' docs/ROADMAP.md
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)